test/main_test.go

```diff
@@ -175,28 +175,38 @@ func deleteNamespaceIfExists(namespace string) {
 	})
 }
 
-func matchingFeature(tags string, features []*gherkin.Feature) bool {
+func matchingFeature(tags string, features []*feature) bool {
 	for _, ft := range features {
-		if len(getMatchingScenarios(tags, ft)) > 0 {
+		if matchesPickles(tags, ft.pickles) {
 			return true
 		}
 	}
 	return false
 }
 
-func showScenarios(features []*gherkin.Feature) {
+func matchesPickles(tags string, pickles []*messages.Pickle) bool {
```

Comment:
According to my above comment, the tags argument should be renamed to "filter" instead?

Reply:
I will change this

Reply:
modified